spec/javascript/components/molecules/AlertBox.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import AlertBox from '~/components/molecules/AlertBox'
+
+describe('AlertBox', () => {
+  test('it renders an alert box with a message', () => {
```

Comment:
one small nit is that I might separate these out into 2 separate tests, so one that is 'it renders an alert box with a message', and one that is 'it calls onClose click when the close button is clicked' or something like that

Reply:
that makes sense! i'll separate them out 👍